exercises/linked-list/.meta/description.md

```diff
@@ -0,0 +1,20 @@
+# Doubly Linked List using `unsafe {}`
+
+Write a doubly linked list and a cursor to traverse it, using unsafe Rust.
+
+The doubly linked list is a fundamental data structure in computer science,
+often used in the implementation of other data structures. They're
+pervasive in functional programming languages, such as Clojure, Erlang,
+or Haskell, but far less common in imperative languages such as Ruby or
+Python.
+
+Each node in a doubly linked list contains data and pointers to the next
+and previous node, if they exist.
+
+New nodes can be efficiently added at any point in the list, if one already has
```

Comment:
It may be worth mentioning practical applications, such as the [dancing links algorithm](https://en.wikipedia.org/wiki/Dancing_Links).